packages/rule-components/src/ReportDetails/RemediatingModal.js

```diff
@@ -0,0 +1,71 @@
+import React from 'react';
+import PropTypes from 'prop-types';
+import { Modal, Button, Tabs, Tab } from '@patternfly/react-core';
+import Markdown from './Markdown';
+
+class RemediatingModal extends React.Component {
+    constructor(props) {
+        super(props);
+        this.state = {
+            isModalOpen: false,
+            activeTabKey: 0
+        };
+    }
+
+  handleModalToggle = () => {
+      this.setState(({ isModalOpen }) => ({
+          isModalOpen: !isModalOpen
+      }));
+  };
+
+  handleTabClick = (event, tabIndex) => {
+      this.setState({
+          activeTabKey: tabIndex
+      });
+  };
+
+  render() {
+      const { reason, resolution, definitions } = this.props;
+      const { isModalOpen, activeTabKey } = this.state;
+
+      return (
+      <>
```

Comment:
I don't really like this usage, it is super easy to overlook it. Can you add `Fragment` instead? You can import it with `Component` as well so you don't have to write `React.Fragment`